include/pmacc/random/methods/AlpakaRand.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2016-2017 Alexander Grund
+/* Copyright 2015-2017 Alexander Grund, Rene Widera
```

Comment:
QUESTION:
Why did you decrease the first year of the license header?

Reply:
This change is coming from the fact that I renamed `Xor.hpp` but git thought I used `XorMIn.hpp` as base. Please ignore this change.